1.8/administration/virtual-networks/isolation.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+nav_title: Isolation
+post_title: Configuring Isolation in Virtual Networks
+menu_order: 20
+---
+
+# iptables rules
+
+DC/OS uses [iptables](http://linux.die.net/man/8/iptables), a fairly high speed, built-in mechanism for filtering traffic in Linux systems, to set up virtual network isolation. We recommend configuring filtering by deploying a homogenous set of rules to all nodes in your infrastructure. In order to simplify this, we also recommend using the [ipset](http://ipset.netfilter.org/ipset.man.html) feature of iptables. 
+
+Set up your own chain that bumps over from the `FORWARD` chain. You can do this by running the following command:
```

Comment:
It looks like "jump" is actually the correct verb: http://www.iptables.info/en/iptables-targets-and-jumps.html